Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -808,8 +813,13 @@ namespace AzToolsFramework
                     }
                     else
                     {
-                        Internal_HandleContainerOverride(
-                            parentUndoBatch, entityId, patch, owningInstance->get().GetLinkId());
+                        // WIP - Re-generate patches to add correct path.
+                        PrefabDom newPatch;
+                        m_instanceToTemplateInterface->GeneratePatch(newPatch, beforeState, afterState);
+                        LinkId linkId = m_prefabFocusInterface->AppendPathFromFocusedInstanceToPatchPaths(newPatch, entityId);
+
+                        // WIP - Store patch to the link closest to the focused prefab.
+                        Internal_HandleContainerOverride(parentUndoBatch, entityId, newPatch, linkId);
```

Comment:
Is this WIP work logged for a follow-up?

Reply:
> do you remember what exactly the WIP or todos here?
Yes, right now editing an entity in a deeply nested prefab will always store the patches to the direct parent instance, regardless of the focus. We need to change this code so that the patches are created relative to the focused prefab, and then stored in the topmost link (between the focused prefab and the direct child instance that owns whatever deeply nested prefab owning the entity that is being edited).

> Also, just to confirm, will these changes keep the same behavior in development until overrides come?
I will double check that nothing sneaked in, but in general these changes should all be behind a setreg flag.
They do change the way things work in the backend slightly (both creating patches and propagation), but as long as the setreg is not set to true they should preserve the same frontend behavior (since the new code should result in the same result for direct child entities of the focused prefab, which are the only ones that the current UI lets you interact with).

Reply:
Created an internal backlog task for this that'll be worked on when override authoring is enabled (lyn-14505)

@chiyenteng , remove the wip comments since its not a good practice to leave them in released code.